deploy/roles/sqitch/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
     - sqitch
 
 - name: Ensure umask is correct
-  sudo: no
+  #sudo: no
```

Comment:
Same here as above.  I think we actually created a separate `umask` role, so perhaps that should just be used?

Reply:
I'm not clear on why we're changing the deploy user's `.bashrc` to set the `umask`.  I reinstated the `sudo: no` as `become: no` (see above).  It might be good to use the `umask` role, but that's a bigger change and I figured I'd go with the quicker fix right now.